radicle-httpd/src/api/v1/projects.rs

```diff
@@ -131,7 +130,7 @@ async fn history_handler(
     };
 
     let storage = &ctx.profile.storage;
-    let repo = storage.repository(project)?;
+    let repo = Repository::open(storage_repo_path(storage, &project))?;
```

Comment:
We try to avoid aliasing as it creates an extra step for the reader to figure out what is being called; I think if you import up to `git::paths`, it should be clear enough: `Repository::open(paths::repository(storage, &project))`

Reply:
I would also almost prefer to move this logic inside context, since it's so common, eg. we could have a function `ctx.surf(&project)`, which would do exactly this.

Reply:
Sounds good, I will remove the aliasing. 

> I would also almost prefer to move this logic inside context, since it's so common, eg. we could have a function ctx.surf(&project), which would do exactly this.

This seems to be an optimization that can be done separately.